.github/workflows/build.yml

```diff
@@ -36,14 +36,16 @@ jobs:
         cd wxWidgets/build_
         sudo make install
     - name: cmake
-      run: CXX=${{ matrix.cxx }} cmake -DCMAKE_BUILD_TYPE=Release .
+      run: CXX=${{ matrix.cxx }} cmake -S . -B _build -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=TreeSheets-relocatable -DTREESHEETS_RELOCATABLE_INSTALLATION=ON
```

Comment:
And also it would hardcode path like `/home/runner/work/aardappel/treesheets/TreeSheets-relocatable` into the program, which would be checked every time, even though it would not exist for users running TreeSheets.

Reply:
What does `relocatable` even mean here? Not familiar..

Reply:
Relocatable means that executing the program will work when moved to another directory. This is typically achieved by referring to resources using paths constructed at runtime by joining relative paths to `dirname(argv[0])`. Filesystem Hierarchy standard kind of implies a fixed installation prefix, which is not very relocatable.

We need to choose:

1. Support both relocatable and non-relocatable installs, look at relative path first, FHS path as a fallback.
2. Support both relocatable and non-relocatable installs, look at FHS path first, relative path as a fallback.
3. Use a configure-time switch between relocatable and non-relocatable installs. This is what the `-DTREESHEETS_RELOCATABLE_INSTALLATION` currently does.
4. Same as 1. but add configure-time switches that allow disabling each of the variants to allow Linux distros to have non-relocatable packages but still allowing PortableApps-style builds.

Reply:
Well, "relocatable" is what it already does, and what it does on other platforms. I don't see why it shouldn't support that, even on Linux. So to me the only question is if it should optionally allow finding things in FHS locations as fallback.